include/drivers/sensor.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright © 2017-2018 The Crust Firmware Authors.
+ * SPDX-License-Identifier: (BSD-3-Clause OR GPL-2.0)
+ */
+
+#ifndef DRIVERS_SENSOR_H
+#define DRIVERS_SENSOR_H
+
+#include <dm.h>
+#include <intrusive.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include <util.h>
+
+#define SENSOR_OPS(dev) \
+	(&container_of((dev)->drv, struct sensor_driver, drv)->ops)
+
+struct sensor_info {
+	const char *const name;       /**< Name exported to SCPI. */
+	const uint8_t     offset;     /**< offset of register. */
+	const int16_t     min_value;  /**< Addition factor. */
```

Comment:
_This_ is the variable that should be named `offset`. :wink:

Reply:
This change has been made